### ISSUE TITLE
Scheduler task for removing unused subscriptions

### ISSUE THREAD

**Issue by fnagel** [fnagel/t3extblog#101]
Add a clean-up scheduler task.

Idea: Integrate this as option in EXT:privacyguard as this should be possible very easily with maximum flexibility.

**Comment by criwe** [comment]
Adding a clean-up scheduler task would be a great idea. 

I always prefer to have one extension (in other words: scheduler integrated in t3extblog), so I have one extension to update, to keep track of and don´t have any stuff I won´t need in this installation. But its just my view.

t3extblog would work without privacyguard, so there would be no real dependency, which would be an argument to put in in privacyguard and keep t3extblog for typical blogfunktions.

Another userful scheduler task would be "delete all deactivated comments which a older than x". Why? You don´t need to delete spam comments by yourself (thats annoying when getting a lot of that ****), just leave them deactivated and wait.

**Comment by fnagel** [comment]
I totally agree to keep everything in one extension but in this case I already maintain an extension which holds pretty much everything needed for this job. 

As this functionality is really a "nice to have" we could make privacyguard an suggested dependency.

Sounds like a good idea. I've added a task list.
